Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1252,7 +1252,15 @@ internal BlockLocator GetBlockLocator(int threshold = 10)
             }
 
             _logger.Debug(
-                "Swapping block chain. (from: {fromChainId}) (to: {toChainId})", Id, other.Id);
+                "Swapping block chain." +
+                " (from: {fromChainId}, TipIndex - {fromTipIndex}, TipHash - {fromTipHash})" +
+                " (to: {toChainId}, TipIndex - {toTipIndex}, TipHash - {toTipHash})",
```

Comment:
To be more compact and consistent with `BlockChain<T>.ReorgedEventArgs` (the terms *old tip* and *new tip*):

```suggestion
                "The blockchain was reorged from " +
                "{OldChainId} (#{OldTipIndex} {OldTipHash}) " +
                "to {NewChainId} (#{NewTipIndex} {NewTipHash}).",
```